lib/async-openai/src/types/chat.rs

```diff
@@ -1008,6 +1008,14 @@ pub struct ChatChoiceLogprobs {
 }
 
 #[derive(ToSchema, Debug, Deserialize, Serialize, Clone, PartialEq)]
+#[derive(Debug, Deserialize, Serialize, Clone, PartialEq)]
```

Comment:
I don't think this was a clean resolution


`ChatChoice` should have kept the existing derive including `ToSchema` in it (this was lost in this commit)

and `StopReason` should only have a single `derive` above it, probably with `ToSchema` in it to relate to the other work in this PR

It also fails to properly lint & compile, ex: `cargo fmt`: https://github.com/ai-dynamo/dynamo/actions/runs/20353715386/job/58484595797?pr=4802